src/options.js

```diff
@@ -1005,6 +1020,41 @@ exports.val = {
   esversion: 5
 };
 
+/**
+ * Unstable options allow control for parsing and linting of proposed additions
+ * to the JavaScript language. Just like the language features they describe,
+ * the presence and behavior of these options is volatile; JSHint reserves the
+ * right to remove or modify them between major version releases.
+ */
+exports.unstable = {
+  /**
+   * Enable parsing support for rest properties for object destructuring
+   * assignment and spread properties for object literals. From the proposal:
+   *
+   * > Rest Properties
+   * >
+   * > Rest properties collect the remaining own enumerable property keys that
+   * > are not already picked off by the destructuring pattern. Those keys and
+   * > their values are copied onto a new object.
+   * >
+   * >     let { x, y, ...z } = { x: 1, y: 2, a: 3, b: 4 };
+   * >     x; // 1
+   * >     y; // 2
+   * >     z; // { a: 3, b: 4 }
+   * >
+   * > Spread Properties
+   * >
+   * > Spread properties in object initializers copies own enumerable
+   * > properties from a provided object onto the newly created object.
+   * >
+   * >     let n = { x, y, ...z };
+   * >     n; // { x: 1, y: 2, a: 3, b: 4 }
+   *
+   * https://github.com/tc39/proposal-object-rest-spread
+   */
+  objspreadrest: true
+};
```

Comment:
I don’t see anything in changes other than tests and error messages where this is actually used. (However, reading this from a phone, so might have missed it)